backend/src/handlers/round.handler.ts

```diff
@@ -0,0 +1,174 @@
+import { Server, Socket } from "socket.io";
+import {
+  enterHostChoosesState,
+  playerChoosePunchlines as playerChoosePunchlinesService,
+  hostChooseWinner as hostChooseWinnerService,
+  enterPlayersChooseState,
+} from "../services/round.service";
+import { ServiceError } from "../util";
+import { RoundState } from "../models/round.model";
+import {
+  assignNextHost,
+  getActivePlayers,
+  initialiseNextRound,
+  isHost,
+} from "./game.handler";
+import { allocatePlayerPunchlines } from "../services/game.service";
+import { SetupType } from "../models";
+
+export default (
+  io: Server,
+  socket: Socket
+): {
+  hostStartRound: (callback: (data: string) => void) => void;
+  playerChoosePunchlines: (
+    punchlines: string[],
+    callback: (data: string) => void
+  ) => Promise<void>;
+  hostViewPunchline: (index: number, callback: (data: string) => void) => void;
+  hostChooseWinner: (
+    winningPunchline: string[],
+    callback: (data: string) => void
+  ) => Promise<void>;
+  hostNextRound: (callback: (data: string) => void) => void;
```

Comment:
Yeah so the reason the default export returns void functions is because it allows me to access the functions initialised inside it for testing 👌🏼 & yeah return void because they all either emit or callback to Socket.IO